_extensions/onlimit-theme/_extension.yml

```
@@ -0,0 +1,35 @@
+title: onlimit-theme
+author: |
+  - Signe Kirk Brødbæk
+  - Daniel B. Ibsen
+  - Luke W. Johnston
+version: 1.0.0
```

```suggestion
version: 0.1.0
```

Let's start with this. I'll add some infra to do continuous delivery/release ☺️